### DISCUSSION TITLE
Is there a way to have the parcel API load dependencies through node_modules?

### DISCUSSION THREAD

**Discussion by matthijsgroen** [parcel-bundler/parcel#8578] [Q&A] [answered]
I have a library that is basically a CLI that uses parcel to generate and build a web-app.

That CLI (own package) has React as dependency.

When I generate a file that uses `import React from "react";` parcel starts to complain that "react" is not a dependency of the host project (the one using the CLI) is there a way that parcel uses the react from the dependencies of the CLI? since the cli is in the dependencies of the host, the 'react' package is in the host's node_modules, but parcel can't find it...

**Comment by mischnic** [comment, marked as answer]
With a resolver plugin, you could retarget react dependencies to be resolved relative to some other arbitrary path (= the CLI).
